src/platform/intel/ace/platform.c

```diff
@@ -52,6 +54,18 @@ static const struct sof_ipc_fw_ready ready
 	.flags = DEBUG_SET_FW_READY_FLAGS,
 };
 
+void platform_panic(uint32_t p)
```

Comment:
Very welcome patch @aborisovich , this is meeting https://github.com/thesofproject/sof/issues/6196 . But, but, is this called, part of the problem is how to hook this (fairly app specific IPC) to Zephyr panic handling. Do you have a solution to this somewhere?

Reply:
No I don't thought about it. This is purely SOF solution...

Reply:
Dropped platform_panic from this PR.